### DISCUSSION TITLE
I got 404

### DISCUSSION THREAD

**Discussion by Yumega** [circled-me/server#6] [Feedback]
stdout: [GIN-debug] Listening and serving HTTP on 0.0.0.0:8080
stdout: [GIN] 2023/10/14 - 14:25:43 | 404 |      33.357µs |  192.168.11.212 | GET      "/"
stderr: 2023/10/14 14:25:43 [DEBUG ERROR]: Status 404, Body: 404 page not found
stdout: [GIN] 2023/10/14 - 14:26:07 | 404 |      22.293µs |  192.168.11.212 | GET      "/"
stderr: 2023/10/14 14:26:07 [DEBUG ERROR]: Status 404, Body: 404 page not found

**Comment by ovizii** [comment]
I am hitting the exact same problem with the default docker-compose.yml

**Comment by nikolaydimitrov** [comment]
Sorry to disappoint you both, but this doesn't have web interface (yet).
The only way to interact with the service is by using the iOS/Android application.

With the app you can share albums on the web, but that's the only web feature at the moment.

**Reply by adamshand** [reply]
> Sorry to disappoint you both, but this doesn't have web interface (yet). The only way to interact with the service is by using the iOS/Android application.

Is there any documentation for the API yet?

**Comment by Yumega** [comment]
@nikolaydimitrov

still error with ios app

notification is login unsuccessful

**Comment by nikolaydimitrov** [comment]
@Yumega that probably means you cannot connect to the server.
Are you using same WiFi on your phone and your server? The right port?
By default it should be connecting to something like `http://192.168.1.34:8080`

**Comment by ovizii** [comment]
> Sorry to disappoint you both, but this doesn't have web interface (yet). The only way to interact with the service is by using the iOS/Android application.
> 
> With the app you can share albums on the web, but that's the only web feature at the moment.

:-) I guess we got ahead of ourselves a bit. Thanks for the pointer, I now could successfully connect the app to my server instance.

**Comment by Yumega** [comment]
@nikolaydimitrov nikolaydimitrov
of course same network 
no, 8080 port used by qbittorent, I changed into 8383. 
Can’t login can’t creat account

**Comment by nikolaydimitrov** [comment]
@Yumega so if you open with your phone browser these pages:
- http://{server-ip}:8080 (qbittorrent)
- http://{server-ip}:8383 (circled-server)

Do you see anything?

If you don't see same error 404 (as in your original post) in the second case, then it's a network access issue. You should see the same logs too.

**Comment by nikolaydimitrov** [comment]
By the way, you should have gotten an iOS prompt to allow local network connections.
Can you make sure it is actually ON? If you go to the "circled.me" Settings screen, you should be seeing this:

<img width="375" alt="image" src="https://github.com/circled-me/server/assets/4488310/f6fb1852-9c2f-4481-9744-854ed0f9f11c">

**Comment by Yumega** [comment]
iphone can visit http://192.168.11.1:8383/  and http://172.38.0.3:8080/ with 404 

the app I have local network enable, but there isn't photos option

**Comment by nikolaydimitrov** [comment]
Thanks for checking. 
Not having photos yet is normal as you haven’t been able to log in I guess. 

And what about the logs from circled-server when you try to create first user?

**Comment by Yumega** [comment]
```
stdout: [GIN] 2023/10/16 - 03:02:15 | 404 |      36.277µs |  192.168.11.212 | GET      "/favicon.ico"
stderr: 2023/10/16 03:02:15 [DEBUG ERROR]: Status 404, Body: 404 page not found
stdout: [GIN] 2023/10/16 - 03:02:40 | 404 |      23.786µs |  192.168.11.213 | GET      "/"
stderr: 2023/10/16 03:02:40 [DEBUG ERROR]: Status 404, Body: 404 page not found
```

ip 212 is my PC, ip 213 is iphone

**Comment by nikolaydimitrov** [comment]
Makes sense, but I can't see any of the requests the app would be making.
Like `/user/login` should be the first one.

**Comment by Yumega** [comment]
error from MySQL

```
stderr: Version: '5.7.43'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
stderr: 2023-10-16T03:01:56.678211Z 3 [Note] Aborted connection 3 to db: 'circled' user: 'root' host: '172.38.0.3' (Got an error reading communication packets)
stderr: 2023-10-16T03:01:56.678211Z 4 [Note] Aborted connection 4 to db: 'circled' user: 'root' host: '172.38.0.3' (Got an error reading communication packets)
stderr: 2023-10-16T09:54:37.654496Z 174 [Note] Aborted connection 174 to db: 'circled' user: 'root' host: '172.38.0.3' (Got an error reading communication packets)
```

**Comment by nikolaydimitrov** [comment]
Can you please try the latest docket-compose example file. Either from the repo or from the readme. 
I updated it couple of days ago but it is possible you have the previous version.

**Comment by Yumega** [comment]
sorry I give up
